packages/flutter/lib/src/widgets/app.dart

```diff
@@ -555,11 +563,18 @@ class WidgetsApp extends StatefulWidget {
   /// This list must not null. Its default value is just
   /// `[const Locale('en', 'US')]`.
   ///
-  /// The order of the list matters. By default, if the device's locale doesn't
-  /// exactly match a locale in [supportedLocales] then the first locale in
-  /// [supportedLocales] with a matching [Locale.languageCode] is used. If that
-  /// fails then the first locale in [supportedLocales] is used. The default
-  /// locale resolution algorithm can be overridden with [localeResolutionCallback].
+  /// The order of the list matters. The default locale resolution algorithm, 
+  /// [basicLocaleListResolution], uses a variation of the truncation algorithm where
```

Comment:
What's the truncation algorithm? Can you link to a definition?

Reply:
I don't think it is a formally defined algo, i'll just remove the explicit name and just describe it.